src/config.js

```diff
@@ -66,17 +72,98 @@ export default class Config {
 	 * @param {*} [value=null] The configuration value. Used if a name is passed to nameOrConfigurations.
 	 */
 	set( name, value ) {
```

Comment:
We can keep `set()`, but there's no use case for it now. Unless... the constructor is using it ;).